src/RowChart.jsx

```diff
@@ -64,18 +64,11 @@ export class RowChart extends React.Component {
     } )
   }
 
-  componentDidMount() {
-    window.addEventListener( 'resize', this._throttledRedraw );
-  }
 
   componentDidUpdate() {
     this._redrawChart()
   }
 
-  componentWillUnmount() {
-    window.removeEventListener( 'resize', this._throttledRedraw );
```

Comment:
Can we get rid of `this._throttledRedraw` in the constructor too?

Reply:
💥  gone 👍